File: Ascii.java
import java.util.Scanner;
public class Ascii{
	public static void main(String[]args){
		Scanner sobjr=new Scanner(System.in);
		System.out.println("Enter any character:");
		char cv=sobjr.nextLine().charAt(0);
		System.out.println("character is=" +cv+ "\n Ascii value is="+ (int)(cv));
	}
}

File: ascii.c
#include<stdio.h>
#include<stdlib.h>
main()
{
	char cv;  //int=char format are not allowed to dwap 
	system("cls");
	printf("Enter any character:");
	scanf("%c",&cv);
	printf("char is =%c ASCII code=%d\n",cv,cv);
	system("pause");
	
}
